__tests__/src/sagas/nations.js

```diff
@@ -0,0 +1,167 @@
+import { call, put, select, takeEvery } from 'redux-saga/effects'
+import { cloneableGenerator } from 'redux-saga/utils'
+import watchProfileUpdate, { checkConnection, createNation, fetchNations, joinNation, leaveNation, getNations } from '../../../src/sagas/nations'
+import {
+  CANCEL_NATION_CREATE, DONE_NATION_CREATE, START_NATIONS_FETCH, DONE_FETCH_NATIONS,
+  NATION_CREATE, CANCEL_LOADING, REQUEST_JOIN_NATION, REQUEST_LEAVE_NATION,
+} from '../../../src/actions/nations'
+import { getPangeaLibrary } from '../../../src/services/container'
+
+jest.mock('BITNATION-Pangea-libs', () => ({
+  eth: {
+    nation: {
+      create: jest.fn(),
+      index: jest.fn(),
+      all: jest.fn(),
+      joinNation: jest.fn(),
+      leaveNation: jest.fn()
+    }
+  }
+}));
+
+jest.mock('react-native-config', () => ({
```

Comment:
Mocks look the same in several files, convention for jest is to have mocks separated from tests in __mocks__ directory (check out this [link](https://facebook.github.io/jest/docs/en/manual-mocks.html)).